xrviz/dashboard.py

```diff
@@ -81,8 +88,13 @@ def create_plot(self, *args):
             if self.var in list(sel_data.coords):  # When a var(coord) is plotted wrt itself
                 sel_data = sel_data.to_dataset(name=f'{sel_data.name}_')
 
+            color_range = {sel_data.name: (sel_data.quantile(colormap_limits[0]),
+                                               sel_data.quantile(colormap_limits[1]))}
+            if color_scale is not 'None':
+                sel_data = getattr(numpy, color_scale)(sel_data)  # Color Scaling
```

Comment:
If we include `1 / x` as a scaling, this will not work. You could have made a simpler dict
```
{
 "linear": lambda x: x,
 "mean": np.mean,
 ...
  "1/x": lambda x: 1 / x
}[color_scale]
```

Reply:
`1/x` is already present as a scaling method using [`numpy.reciprocal`](https://docs.scipy.org/doc/numpy/reference/generated/numpy.reciprocal.html) and is available as `reciprocal` in color_scale options.

Reply:
My mistake :)
It's not the name I would have guessed for it.

Note that you are calling numpy functions here, but actually they get converted to xarray functions ([example](http://xarray.pydata.org/en/stable/generated/xarray.DataArray.mean.html)). I don't know where reciprocal is implemented in xarray.